runtime-decompiler/src/test/java/org/jrd/backend/data/CliTest.java

```diff
@@ -64,6 +67,21 @@ static void startup() {
         );
     }
 
+    private static String findFreshAgent() throws IOException {
+        String cwd = System.getProperty("user.dir");
+        File agentDir = new File(cwd + File.separator + ".." + File.separator + "decompiler_agent" + File.separator + "target");
```

Comment:
Just rememory from todays discussion:  The freshlpy build agent is searched also in https://github.com/pmikova/java-runtime-decompiler/pull/177

We agreed to push both, and later removed duplicated code.